docs/Makefile

```diff
@@ -37,7 +37,6 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)
-	-rm -rf api
```

Comment:
This was just so it wouldn't clobber the api/ folder that i had put in.  I'll add this back in with the new autoapi now that it's working